docs/siem-apis.asciidoc

```diff
@@ -5,7 +5,7 @@
 
 You can use these APIs to interface with {es-sec} features:
 
-NOTE: Console supports only Elasticsearch APIs. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to {kibana-ref}/console-kibana.html[Run {es} API requests].
+NOTE: Console supports sending requests to {kib} APIs: prepend any {kib} API endpoint with `kbn:` and send the request via Console, for example, `GET kbn:/api/index_management/indices`.
```

Comment:
```suggestion
NOTE: Console supports sending requests to {kib} APIs. Prepend any {kib} API endpoint with `kbn:` and send the request via Console. For example, `GET kbn:/api/index_management/indices`.
```

Reply:
In addition to @jmikell821's suggestion above, I'd also recommend a line break to put the sample request all on one line:
```suggestion
NOTE: Console supports sending requests to {kib} APIs. Prepend any {kib} API endpoint with `kbn:` and send the request via Console. For example:
`GET kbn:/api/index_management/indices`
```